_source/logzio_collections/_log-sources/dotnet.md

```diff
@@ -614,5 +615,128 @@ For the example above, you'd use `MyAppLogzioAppender`.
 </div>
 <!-- tab:end -->
 
+<!-- tab:start -->
+<div id="serilog">
+
+#### Configure Serilog
+  
+<!-- info-box-start:info -->
+This integration is based on [Serilog.Sinks.Logz.Io repository](https://github.com/serilog-contrib/Serilog.Sinks.Logz.Io).
+{:.info-box.note}
```

Comment:
can you add that for more complex usage and settings information they can go to this repo?